Home Work 11 (html css advanced)

GeorgyChirkov opened the pull request.

Commit:
complete: task-03

Commit:
fix: minor test issues

aneelia commented on src/ex11_html-css-advanced/task-03.html:
```diff
@@ -0,0 +1,38 @@
+                    <div class="active">Active tasks: &lt;N&gt;</div>
+                    <div class="finished">Finished tasks: &lt;M&gt;</div>
+                </div>
+                <div class="author">Kanban board by &lt;NAME&gt;, &lt;YEAR&gt;</div>
```
ну тут свое имя можно написать :)

aneelia commented on src/ex11_html-css-advanced/task-03.html:
```diff
@@ -0,0 +1,38 @@
+                    <img src="images/arrow-down.svg" alt="arrow-down" id="arrow">
+                </div>
+            </div>
+        </header>
```
просто так хедер и футер не оставляют,  нужно хотя бы заглушку сделать - например, пустой main

aneelia commented on src/ex11_html-css-advanced/task-03-2.css:
```diff
@@ -0,0 +1,34 @@
+footer {
+    background: #0067A3;
+    width: 100%;
```
на flex можно попробовать заменить

aneelia reviewed (commented):
good job!

Commit:
fix: minor

Commit:
complete: task-01

Commit:
fix: change secont main elem into section

Commit:
complete: task-02

aneelia commented on src/ex11_html-css-advanced/task-01.css:
```diff
@@ -0,0 +1,93 @@
+    animation: spin 1s linear infinite;
+  }
+  
+  @keyframes spin {
```
название так себе

aneelia commented on src/ex11_html-css-advanced/task-01.css:
```diff
@@ -0,0 +1,93 @@
+    }
+}
+/* animation 3 */
+.inf {
```
что есть инф

aneelia commented on src/ex11_html-css-advanced/task-01.css:
```diff
@@ -0,0 +1,93 @@
+    100% { transform: rotate(360deg); }
+  }
+/* animation 2 */
+.task2 {
```
названия с task все переделать

aneelia commented on src/ex11_html-css-advanced/task-03-1.css:
```diff
@@ -0,0 +1,63 @@
+    flex-direction: row;
+    align-items: center;
+}
+#logo {
```
без id никак?

aneelia commented on src/ex11_html-css-advanced/task-03-1.css:
```diff
@@ -0,0 +1,63 @@
+    color: #FFFFFF;
+    margin-left: 8px;
+}
+.elements {
```
в множественном числе обычно не называют
element-group

aneelia commented on src/ex11_html-css-advanced/task-03-1.css:
```diff
@@ -0,0 +1,63 @@
+    background: #FFFFFF;
+    border-radius: 5px;
+}
+.add-text {
```
это кнопка
курсор добавить
название изменить

aneelia commented on src/ex11_html-css-advanced/task-03-1.css:
```diff
@@ -0,0 +1,63 @@
+    flex-direction: row;
+    align-items: center;
+}
+#avatar {
```
дальше будет требование без id, лучше сразу переделай

aneelia commented on src/ex11_html-css-advanced/task-03-2.css:
```diff
@@ -0,0 +1,34 @@
+footer {
```
зачем это два файла?

aneelia commented on src/ex11_html-css-advanced/task-03-2.css:
```diff
@@ -0,0 +1,34 @@
+    line-height: 21px;
+    color: #FFFFFF;
+}
+.footer-text-left {
```
лучше смысловое дать название, а не левый текст

aneelia commented on src/ex11_html-css-advanced/task-03-2.css:
```diff
@@ -0,0 +1,34 @@
+.active {
+    margin-left: 67px;
+}
+.finished {
```
не, так не пойдет
надо сделать общий контейнер и там разносить флексом активные и закрытые задачи, а не маржинами

Commit:
Merge branch 'master' of https://github.com/GeorgyChirkov/external-courses into HW11

Commit:
fix: bugs, semantic, counf of files

fixed bad naming of classes
deleted ids and replaced it to classes
fix semantic, delete section add change main place
join css files to one

Commit:
fix: replace ids to classes
